.Documentation/articles/innoactive-creator/suspending-interactions.md

```diff
@@ -0,0 +1,7 @@
+# Suspending Interactions
+
+You can let your trainees to interact with a [training object](training-scene-object.md) by adding certain properties to it. By default, objects stay interactive throughout the whole training course: a trainee could push a button and flick a switch at any time. The training feels more real, but it can turn chaotic in complex courses and scenes. To prevent that, you can temporarily disable interactions for a property, and restore it only when your trainee needs it.
+
+To disable user interactions with all properties from the start, attach the `LockObjectsOnSceneStart` component to any active object in the scene. You can suspend user interactions for individual training properties by using the `Lock` behavior. 
```

Comment:
I think the `Lock` behavior is now [obsoleted](https://github.com/Innoactive/Basic-Conditions-And-Behaviors/blob/1453c3e3bbb42dfaffc7634a114353211e54e22d/Runtime/Behaviors/LockObjectBehavior.cs#L12) 🤔